src/components/formik-toggle/FormikToggle.tsx

```diff
@@ -11,58 +11,61 @@ type Props = {
     name: string;
   };
   form: {
-    ['errors']: string;
     ['touched']: boolean;
   };
   disabled: boolean;
   validationMessage: string;
   label: string;
   placeholderText: string;
-  error: boolean;
   tooltipToggle?: any;
   focussed: boolean;
   iconRight?: any;
 };
 
-
 const styleVariables = loadStyleVariables();
 
-const Toggle = styled.label`
+const Toggle = styled.div`
   position: relative;
   display: inline-block;
   width: 40px;
   height: 20px;
-`;
-
-const Slider = styled.span<{
-  error?: boolean;
-  disabled?: boolean;
-  focussed?: boolean;
-}>`
-  position: absolute;
-  cursor: pointer;
-  top: 0;
-  left: 0;
-  right: 0;
-  bottom: 0;
-  content: "o";
-  border: ${props =>
-    props.error
-      ? `2px solid ${styleVariables.redGoogle}`
-      : `2px solid ${styleVariables.greenDark}`};
-  border-radius: 30px;
-  transition: all 0.4s;
-
-  background: ${props =>
-    props.error ? `${styleVariables.red}` : `${styleVariables.green}`};
-  &:checked {
-    transform: translateX(20px);
-    background-color: ${styleVariables.red};
+  span {
+    position: absolute;
+    cursor: pointer;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    border: 2px solid ${styleVariables.gray};
+    color: white;
+    background: ${styleVariables.grayLight};
+    border-radius: 30px;
+    transition: all 0.4s;
+    :after {
+      top: 2px;
+      left: 5px;
+      font-size: 10px;
```

Comment:
Tip: altijd font-size variabelen gebruiken om een wildgroei aan font-sizes te voorkomen. Maar in dit geval gebruik je de o als icoon, dus mag in dit geval wel :D